packages/helper-backend/src/langflow-mapping/langflow-mapping.controller.ts

```diff
@@ -23,11 +38,39 @@ export class LangflowMappingController {
   }
 
   @Get()
+  @ApiOperation({
+    summary: 'Get all mappings',
+    description: 'Retrieves all available Langflow mappings with complete configuration details'
+  })
+  @ApiResponse({
+    status: 200,
+    description: 'List of all mappings with their configurations',
+    type: [LangFlowMapping],
+    schema: {
```

Comment:
The schema should be handled by `ApiProperty` annotations on the `LangFlowMapping` class